src/components/SingleBook.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable camelcase */
```

Comment:
- [ ] [REQUIRED] Kindly fix linter errors instead of disabling them. This is important in keeping high code standards.